_pytest/runner.py

```diff
@@ -83,6 +86,16 @@ def runtestprotocol(item, log=True, nextitem=None):
         item.funcargs = None
     return reports
 
+def show_test_item(item):
+    """Show test function, parameters and the fixtures of the test item."""
+    tw = item.config.get_terminal_writer()
+    tw.line()
+    tw.write(' ' * 8)
+    tw.write('{0}'.format(item._nodeid))
```

Comment:
Actually it's even the same as `item._noteid` because node id is a string. Fixed.